lib/phoenix_token_auth/controllers/sessions_controller.ex

```diff
@@ -15,8 +15,15 @@ defmodule PhoenixTokenAuth.Controllers.Sessions do
   Responds with status 200 and {token: token} if credentials were correct.
   Responds with status 401 and {errors: error_message} otherwise.
   """
+  def create(conn, %{"username" => username, "password" => password, "grant_type" => "password"}) do
```

Comment:
please remove the `grant_type` param. it's not really relevant and without it, people not using ember can use this as well.